lib/hexpm/pwned/hexpm.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Hexpm.Pwned.Hexpm do
```

Comment:
nitpick: what do you think about calling this `Hexpm.Pwned.HaveIBeenPwned`? 

We have a similar module `Hexpm.Billing.Hexpm` but the idea there was it's using `https://billing.hex.pm` as opposed to a local fake adapter.

Reply:
oh yes, good idea ;)